components/SpotlightShare/SpotlightShare.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FunctionComponent } from 'react';
+import Modal from 'react-modal';
+import { Button } from '../Button';
+import classNames from 'classnames';
+import { FaFacebookF, FaTwitter } from 'react-icons/fa';
+import { MdEmail } from 'react-icons/md';
+import './styles.css';
+
+interface SpotlightShareProps {
+  maxHeight?: string;
+  minHeight?: string;
+  className?: string;
+}
+const customStyles = {
+  content: {
+    top: '50%',
+    left: '50%',
+    width: '50%',
+    height: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)'
+  }
+};
+
+const SpotlightShare: FunctionComponent<SpotlightShareProps> = props => {
+  const [modalIsOpen, setIsOpen] = React.useState(false);
+  const openModal = () => {
+    setIsOpen(true);
+  };
+  const closeModal = () => {
+    setIsOpen(false);
+  };
+  return (
+    <div className={classNames(props.className)}>
```

Comment:
If you're only going to provide a single className, you don't need to use `classNames`.